text/conversations/00_dyrwood/00_cv_lord_harond.stringtable

```diff
@@ -480,7 +480,7 @@ Er dreht sich zu Aelys um. "Lass uns gehen, mein Liebling. Schnell."</DefaultTex
     </Entry>
     <Entry>
       <ID>255</ID>
-      <DefaultText>Ein Elfenmann läuft in einer Ecke des Raumes auf und ab und kaut am Daumen seines edlen Seidenhandschuhs. "Was hast du hier oben verloren? Es sei denn ... hast du Neuigkeiten von meiner Tochter? Ich muss sie aus diesem armseligen Dorf fortbringen ..." Er läuft weiter auf und ab.</DefaultText>
+      <DefaultText>Der Elfenmann läuft in einer Ecke des Raumes auf und ab und kaut am Daumen seines edlen Seidenhandschuhs. "Was hast du hier oben verloren? Es sei denn ... hast du Neuigkeiten von meiner Tochter? Ich muss sie aus diesem armseligen Dorf fortbringen ..." Er läuft weiter auf und ab.</DefaultText>
```

Comment:
Irgendwie schon, vorher war es halt extrem unpersönlich, obwohl man dem direkt gegenüberstand. Bewegen tun sie sich ja nicht, weil das Spiel in Gesprächen immer in den Pausemodus geht …, mal sehen ob das im Addon kommt.